client/my-sites/plan-features/style.scss

```diff
@@ -95,7 +96,7 @@ $plan-features-sidebar-width: 272px;
 	border-left: solid 1px lighten( $gray, 27% );
 	background-color: lighten( $gray, 35% );
 	position: relative;
-	width: 300px;
+	width: 25%;
```

Comment:
I'm not sure about this. We only show three columns in the `/plans/:site` page from the sidebar link. However I've done some cross-browser testing and this seems ok in chrome/firefox/ie11. So I guess if the percentages for table columns don't add up to 100%, the browser just uses them as proportions of the whole (i.e., 25/75 = 1/3)?